### PR TITLE
docs: link from install docs to create-next-app-docs

### PR DESCRIPTION
**Pull request by Mutesa-Cedric**
as many developers are moving to bun, its a good idea that it also be included as an installation method.

### CODE REVIEW

**Review by styfle — CHANGES_REQUESTED** [review]
We already updated the `create-next-app` docs so lets link there instead

https://nextjs.org/docs/app/api-reference/create-next-app

File: docs/01-getting-started/01-installation.mdx [hunk 1]
```diff
@@ -12,7 +12,7 @@ System Requirements:
 
 - [Node.js 16.14](https://nodejs.org/) or later.
 - macOS, Windows (including WSL), and Linux are supported.
-
+.
 ## Automatic Installation
 
 We recommend starting a new Next.js app using `create-next-app`, which sets up everything automatically for you. To create a project, run:
```
Comment (styfle) [inline docs/01-getting-started/01-installation.mdx:18]:
```suggestion
We recommend starting a new Next.js app using [`create-next-app`](/docs/app/api-reference/create-next-app), which sets up everything automatically for you. To create a project, run:
```

File: docs/01-getting-started/01-installation.mdx [hunk 2]
```diff
@@ -21,6 +21,14 @@ We recommend starting a new Next.js app using `create-next-app`, which sets up e
...
 ```
 
+Alternatively, if you prefer to use bun for creating a Next.js application, you can do so by running:
+
+```bash filename="Terminal"
+bun create next-app
+```
+
+For this to work, you need to have bun installed on your system
+
```
Comment (styfle) [inline docs/01-getting-started/01-installation.mdx:31]:
```suggestion
```

**Review by styfle — DISMISSED** [review]

File: docs/01-getting-started/01-installation.mdx [hunk 3]
```diff
@@ -12,10 +12,10 @@ System Requirements:
 
 - [Node.js 16.14](https://nodejs.org/) or later.
 - macOS, Windows (including WSL), and Linux are supported.
-
+.
```
Comment (leerob) [inline docs/01-getting-started/01-installation.mdx:15]: This needs to be removed

**Review by balazsorban44 — APPROVED** [review]

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: f5e0822006b2cad04c40e2b3674c63cf22980029

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: f5e0822006b2cad04c40e2b3674c63cf22980029

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Pull request merged**